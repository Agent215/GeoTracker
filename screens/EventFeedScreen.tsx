
import * as React from 'react';

import { StyleSheet } from 'react-native';

import { Text, View } from '../components/Themed';

import { Image, ScrollView } from 'react-native';





export default function EventFeedScreen() {
  return (

    <View style={styles.container}>
      <ScrollView>
        <Text style={{ fontSize: 96 }}>Demo 1</Text>
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Text style={{ fontSize: 60 }}>Try and scroll!</Text>
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Text style={{ fontSize: 60 }}>Scroll down</Text>
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Text style={{ fontSize: 60 }}>Scroll down</Text>
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Image source={logo} />
        <Text style={{ fontSize: 60 }}>Scroll back up!</Text>
      </ScrollView>
    </View>
  );
}

const logo = {
  uri: 'https://reactnative.dev/img/tiny_logo.png',
  width: 64,
  height: 64
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    alignItems: 'center',
    justifyContent: 'center',
  },
  title: {
    fontSize: 20,
    fontWeight: 'bold',
  },
  separator: {
    marginVertical: 30,
    height: 1,
    width: '80%',
  },
});