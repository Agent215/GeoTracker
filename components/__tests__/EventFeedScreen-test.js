import * as React from 'react';
import renderer from 'react-test-renderer';

import EventFeedScreen from '../../screens/EventFeedScreen';

it(`EventFeedScreen renders correctly`, () => {
  const tree = renderer.create(<EventFeedScreen/>).toJSON();

  expect(tree).toMatchSnapshot();
});
