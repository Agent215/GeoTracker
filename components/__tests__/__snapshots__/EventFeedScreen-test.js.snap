// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventFeedScreen renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <RCTScrollView>
    <View>
      <Text
        style={
          Array [
            Object {
              "color": "#000",
            },
            Object {
              "fontSize": 96,
            },
          ]
        }
      >
        Demo 1
      </Text>
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#000",
            },
            Object {
              "fontSize": 60,
            },
          ]
        }
      >
        Try and scroll!
      </Text>
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#000",
            },
            Object {
              "fontSize": 60,
            },
          ]
        }
      >
        Scroll down
      </Text>
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#000",
            },
            Object {
              "fontSize": 60,
            },
          ]
        }
      >
        Scroll down
      </Text>
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Image
        source={
          Object {
            "height": 64,
            "uri": "https://reactnative.dev/img/tiny_logo.png",
            "width": 64,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#000",
            },
            Object {
              "fontSize": 60,
            },
          ]
        }
      >
        Scroll back up!
      </Text>
    </View>
  </RCTScrollView>
</View>
`;
